### ISSUE TITLE
using pre training generated for albert for Electra

### ISSUE THREAD

**Issue by 008karan** [google-research/electra#6]
I have pretraining data generated for Albert. Can I use it for in pretraining of Electra? 
As I am doing it for Hindi language, Using sentence piece vocab will suffice?

**Comment by clarkkev** [comment]
I am not totally sure, because I don't know all the details of ALBERT's code. I think ALBERT does the masking when creating the data rather than on-the-fly. So to run it with ELECTRA you would either have to

1. Re-create the data with no masking.
2. Remove `masked_inputs = pretrain_helpers.mask(...` from `run_pretraining.py`
3. Change that line to`masked_inputs = pretrain_helpers.mask(config, pretrain_helpers.unmask(pretrain_data.features_to_inputs(features)), config.mask_prob)`, which un-masks the input and then re-masks it with dynamic masking.

Out of those, 1 or 3 would be better because dynamic masking helps results a bit (see Table 1 of [the RoBERTa paper](https://arxiv.org/pdf/1907.11692.pdf).

However, I am not sure if there are other incompatibilities with the tfrecord format that would still cause errors. Also, you would still run into issues if you want to use our code for fine-tuning the model after it is pre-trained because we use different tokenization code than ALBERT.

**Comment by SantoshGuptaML** [comment]
I always wanted to see the Electra type training on the Albert architecture
